### PR TITLE
Update DG v1.3

### PR DESCRIPTION
**Pull request by laihuiqi**
Update developer guide on the implementation of feature `clear_by`.

### CODE REVIEW

**Comment by laihuiqi** [comment]
Yes, you are right. Thank you for pointing it out!

File: src/main/java/seedu/address/logic/LogicManager.java [hunk 1]
```diff
@@ -47,7 +50,14 @@ public CommandResult execute(String commandText) throws CommandException, ParseE
...
         try {
-            storage.saveAddressBook(model.getAddressBook());
+            if (commandResult.getType() == TodoType.NONE) {
+                storage.saveAddressBook(model.getAddressBook());
+            } else if (commandResult.getType() == TodoType.TODO) {
+                storage.saveTodoList(model.getTodoList());
+            } else if (commandResult.getType() == TodoType.NOTE) {
+                storage.saveNoteList(model.getNoteList());
+            }
+
```
Comment (jiasheng59) [inline src/main/java/seedu/address/logic/LogicManager.java:60]: An elegant way of handling storage 👍

File: src/main/java/seedu/address/logic/commands/task/note/ClearNoteCommand.java [hunk 2]
```diff
@@ -6,31 +6,34 @@
...
+import seedu.address.model.tag.TodoType;
+import seedu.address.model.task.Note;
 
 /**
  * Clears the address book.
  */
 public class ClearNoteCommand extends Command {
 
-    public static final String COMMAND_WORD = "clear note";
+    public static final String COMMAND_WORD = "clear_note";
```
Comment (jiasheng59) [inline src/main/java/seedu/address/logic/commands/task/note/ClearNoteCommand.java:19]: I think having an underscore making COMMAND_WORD into a single word is a good decision and should be applied to all commands also. 👍

File: src/main/java/seedu/address/logic/commands/task/todo/EditDeadlineCommand.java [hunk 3]
```diff
@@ -28,12 +29,14 @@ public class EditDeadlineCommand extends Command {
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Edits the deadline of the specified todo from current available list.\n"
             + "Parameters: INDEX (INDEX must be a positive integer) "
-            + "[" + PREFIX_DEADLINE + "DEADLINE] (must be in format dd-mm-yyyy)\n"
+            + "[" + PREFIX_DEADLINE + "DEADLINE] (must be in format yyyy-mm-dd)\n"
```
Comment (jiasheng59) [inline src/main/java/seedu/address/logic/commands/task/todo/EditDeadlineCommand.java:32]: I actually find this a bit confusing, cuz in `InterviewDate` class, we are using a different format which is `MM/dd/yyyy hh:mm a`. Will it be more consistent to have a single format for all date-time? @jianminglok @benjamin-wee @zm-l how do yall think?

**Review by jiasheng59 — COMMENTED** [review]
Overall the code looks neat and clean! It LGTM and should be merged asap since this involves a massive GUI changes also

**Pull request merged**